TemplateApplication/Onboarding/HealthKitPermissions.swift

```diff
@@ -11,6 +11,14 @@ import SpeziOnboarding
 import SwiftUI
 
 
+/// Gain access to HealthKit data.
+///
+/// Customize the HealthKitPermissions view to specify your app's usage of HealthKit data to the user.
+///
+/// You can learn more about [Spezi Onboarding in the GitHub repository](https://github.com/StanfordSpezi/SpeziOnboarding)
+///
+/// ![A screenshot of the HealthKitPermissions screen](HealthKitAccess)
+/// ![A screenshot of the HealthKitSheet screen](HealthKitSheet)
```

Comment:
The screenshots are currently shown in a really large size; it might not be the desirable size. I would suggest that we use a trick of having a `@Row` with a size of e.g., `3` columns that only contains one or two and therefore scales the images accordingly.
```suggestion
/// @Row(numberOfColumns: 3) {
///     @Column(size: 1) {
///         @Image(source: "HealthKitAccess", alt: "HealthKit Onboarding Flow")
///     }
///     @Column(size: 1) {
///         @Image(source: "HealthKitSheet", alt: "Permissions screen of the HealthKit framework")
///     }
/// }
```

Applies to other parts of the documentation as well.